srcpkgs/lz4/template

```diff
@@ -1,21 +1,22 @@
 # Template file for 'lz4'
 pkgname=lz4
-reverts="131_1 130_1 129_1 128_1 127_1 126_1 125_1 124_1 123_1 122_1"
-version=1.9.4
+version=1.10.0
 revision=1
-bootstrap=yes
-build_style=gnu-makefile
-make_check_target=test
+bootstrap="yes"
+build_style="gnu-makefile"
+make_check_target="test"
 checkdepends="python3"
-short_desc="LZ4 compression utilities"
-maintainer="Orphaned <orphan@voidlinux.org>"
-license="BSD-2-Clause, GPL-2.0-or-later"
+short_desc="Extremely fast compression algorithm"
+maintainer="Will Reed <wreed@disroot.org>"
+# all files under source 'lib' directory are BSD-2, others are GPLv2+
+license="BSD-2-Clause AND GPL-2.0-or-later"
 homepage="https://lz4.github.io/lz4"
-changelog="https://raw.githubusercontent.com/lz4/lz4/dev/NEWS"
-distfiles="https://github.com/lz4/lz4/archive/v${version}.tar.gz"
-checksum=0b0e3aa07c8c063ddf40b082bdf7e37a1562bda40a0ff5272957f3e987e0e54b
+changelog="https://github.com/lz4/lz4/raw/refs/tags/v${version}/NEWS"
```

Comment:
changelogs should not come from tags when possible, as it is useful to know what's changed without needing *that version* of the package